projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -266,7 +266,8 @@ public function add_options_menu( $wp_admin = false ) {
 
 		add_submenu_page( 'options-general.php', esc_attr__( 'Hosting Configuration', 'jetpack' ), __( 'Hosting Configuration', 'jetpack' ), 'manage_options', 'https://wordpress.com/hosting-config/' . $this->domain, null, 6 );
 
-		$this->hide_submenu_page( 'options-general.php', 'sharing' );
+		// We are safe to remove this page because it does not have any additional information and it's not a main page.
+		remove_submenu_page( 'options-general.php', 'sharing' );
```

Comment:
Should we do the same for Atomic sites?

With an `editor` user, I get a "Sharing" menu that has no link
![](https://cln.sh/92ySkv+)

Reply:
If we'll remove it for Atomic, should we also remove it from Jetpack sites? 🤔  If so, we could move that snippet directly to class-admin-menu.

Reply:
I think we should because it's already removed in WPCOM. I'll update the PR right now! :)